web/app/themes/justicejobs/single-agency.php

```diff
@@ -67,145 +67,6 @@ class="hero__img hero__img--mobile"
-                                            </svg>
-                                        </button>
-                                    </div>
-                                </div>
-                            <?php } ?>
-                        <?php }
-                    }
-                    ?>
-                    <!-- / END TOP LEFT VIDEO -->
-                    <!-- TOP RIGHT CAROUSEL -->
-                    <div class="agency__carousel accessible-carousel"
-                         style="background-color: <?php the_field('agency_colour'); ?>;">
-                        <?php if (have_rows('roles_carousel')) : ?>
-                            <h3 class="heading--xs"><?php the_field('carousel_title'); ?></h3>
-                            <div id="accessible-carousel-latest-roles">
-                                <ul class="accessible-carousel__container">
-                                    <?php while (have_rows('latest_roles_carousel')) :
-                                        the_row();
-                                        $agency_name = get_sub_field('agency_name');
-                                        $position_title = get_sub_field('position_title');
-                                        $position_location = get_sub_field(' position_location');
-                                        $more_link = get_sub_field('find-out-more_link');
-                                        ?>
-                                        <li class="accessible-carousel__slide">
-                                            <p class="heading--xxs"><?= $agency_name; ?></p>
-                                            <h3 class="heading--sm"><?= $position_title; ?></h3>
-                                            <p class="heading--xxs"><?= $position_location; ?></p>
-                                            <?php if (isset($more_link['url'])) : ?>
-                                                <a href="<?php echo esc_url($more_link['url']); ?>"
-                                                   target="<?php echo $more_link['target'] ?? ''; ?>"
-                                                   class="btn-secondary btn-secondary--light">
-                                                    <?php echo esc_html($more_link['title']); ?>
-                                                    <svg width="8" height="13">
-                                                        <use xlink:href="#icon-arrow"></use>
-                                                    </svg>
-                                                </a>
-                                            <?php endif ?>
-                                        </li>
-                                    <?php endwhile; ?>
-                                </ul>
-                            </div>
-                        <?php endif; ?>
-                    </div>
-                    <!-- / END TOP RIGHT CAROUSEL -->
-                <?php endif ?>
-
```

Comment:
There was a lot of text deleted from this file. What was the desired affect and has functionality been taken away? 
Thank you

Reply:
The top row of the agency page blocks currently is broken. It requires a video. Josh has not used it due to this. We have decided as quick fix for content handover d to remove the top row section. With a plan to review the agency blocks in the future so they are more flexible. I removed the code as we can always get it back in GIT. But I can comment it out if you like.

Reply:
So to answer your question yes we are taking away functionality for now. As its not used and will complicate content training handover.

Reply:
Great, thanks for the explanation :)